lib/rucio/core/replica.py

```diff
@@ -61,6 +65,7 @@
 from rucio.common.types import InternalScope
 from rucio.common.utils import chunks, clean_surls, str_to_date, add_url_query
 from rucio.core.config import get as config_get
+from rucio.core.config import read_from_cache, write_to_cache
```

Comment:
It's only a minor detail, but it would be better not to use these methods. (They should not be public, but internal to `core.config` anyway. 
The issue with this is that you are basically using the cache region of the config module. In principle this works, but it may have some side-effects. It would be better to establish a REGION directly in core/replica.

Reply:
OK. Fixed.